implementations/rust/ockam/ockam/src/remote.rs

```diff
@@ -283,32 +283,28 @@ mod test {
     #[allow(non_snake_case)]
     #[ockam_macros::test]
     async fn forwarding__ephemeral_address__should_respond(ctx: &mut Context) -> Result<()> {
-        let cloud_address;
-        if let Some(c) = get_cloud_address() {
-            cloud_address = c;
-        } else {
-            ctx.stop().await?;
-            return Ok(());
-        }
+        let cloud_address = match get_cloud_address() {
```

Comment:
I know clippy moans about this but if I remember correctly we ended up disabling this rule in CI by popular request as `if let` is usually a lot more readable.

I don't feel strongly about this so it shouldn't block this PR but if you also prefer the other style you can run the linter locally without the rule using something like:

`RUSTFLAGS="-A clippy::let-unit-value" ./gradlew :rust:lint`

or even:

`cargo clippy --no-deps -- -D warnings -A clippy::let-unit-value`

Reply:
Thanks for the info. Actually, the clippy message I was getting was `needless_late_init` (see below).

I've updated the PR to use `let cloud_address = if let Some(c) = get_cloud_address() { ... } else { ... }`. I've noticed that pattern is used elsewhere in Ockam; I hope it's okay. 

```
error: unneeded late initalization
   --> implementations/rust/ockam/ockam/src/remote.rs:286:9
    |
286 |         let cloud_address;
    |         ^^^^^^^^^^^^^^^^^^
    |
    = note: `-D clippy::needless-late-init` implied by `-D warnings`
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#needless_late_init
help: declare `cloud_address` here
    |
287 |         let cloud_address = if let Some(c) = get_cloud_address() {
    |         +++++++++++++++++++
help: remove the assignments from the branches
    |
288 -             cloud_address = c;
288 +             c
    |
help: add a semicolon after the `if` expression
    |
292 |         };
    |          +
```